lib/gears/color.lua

```diff
@@ -338,6 +338,11 @@ end
 -- @treturn string color if it is valid, else fallback.
```

Comment:
maybe mention in docs what it skip checking the certain "types" of colors?

Reply:
Sorry, what?

Reply:
my bad, i didn't noticed `%x` in the regexp so i thought it's only checking the length and if length seems like argb value it just returns it back